endoreg_db/views/Frames_NICE_and_PARIS_classifications_views.py

```
@@ -1,61 +1,38 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework import status
-from ..models import VideoFile
-# from ..serializers.Frames_NICE_and_PARIS_classifications import ForNiceClassificationSerializer
+from endoreg_db.models import VideoFile
+from ..serializers._old.Frames_NICE_and_PARIS_classifications import (
+    ForNiceClassificationSerializer,
+    ForParisClassificationSerializer
+)
+
 import logging
```

Ruff (0.8.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)